otx/cli/utils/multi_gpu.py

```diff
@@ -284,6 +305,11 @@ def _spawn_multi_gpu_processes(self, output_path: str) -> List[mp.Process]:
             task_p.start()
             processes.append(task_p)
 
+        if origin_cuda_visible_devices is None:
+            del os.environ["CUDA_VISIBLE_DEVICES"]
+        else:
+            os.environ["CUDA_VISIBLE_DEVICES"] = origin_cuda_visible_devices
```

Comment:
I'm just curious, why do we need this?

Reply:
To apply `CUDA_VISIBLE_DEVICES` environment variable to child process, `CUDA_VISIBLE_DEVICES` should be changed before spawning child process in parent process. So, this code is for reverting `CUDA_VISIBLE_DEVICES` value as before `CUDA_VISIBLE_DEVICES` is modified.